test/libsolidity/syntaxTests/inheritance/base_arguments_multiple_inheritance.sol

```diff
@@ -0,0 +1,8 @@
+contract Base {
+    constructor(uint) public { }
+}
+contract Base1 is Base(3) {}
+contract Derived is Base, Base1 {
+    constructor(uint i) Base(i) public {}
```

Comment:
Will this create an error in a future PR?

Reply:
Ah, ok - I took this from a failing end-to-end test and suspected that this should actually *not* be an error (which is why I stopped searching the inheritance tree above in the first place). But yes, this should, of course, be an error as well - I'll change this PR accordingly.